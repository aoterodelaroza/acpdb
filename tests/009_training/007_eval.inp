## check: 007_eval.out -a1e-10
## delete: 007_eval.db
## delete: 007_eval-set.h2o_h2o_1.xyz
## delete: 007_eval-set.h2o_h2o_2.xyz
## delete: 007_eval-set.h2o_h2o.xyz
## delete: 007_eval-set.nh3_nh3_1.xyz
## delete: 007_eval-set.nh3_nh3_2.xyz
## delete: 007_eval-set.nh3_nh3.xyz
## labels: regression quick

system rm -f 007_eval.db
connect 007_eval.db

insert method m_empty
end
insert method m_ref
end
insert set s22b
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert set s22b-b
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert evaluation
 method m_empty
 property s22b.nh3_nh3
 value 0.2
end
insert evaluation
 method m_empty
 property s22b.h2o_h2o
 value 0.3
end
insert term
 method m_empty
 property s22b.h2o_h2o
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property s22b.h2o_h2o
 atom H
 L 0
 exponent 0.2
 value 0.001
end
insert term
 method m_empty
 property s22b.nh3_nh3
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property s22b.nh3_nh3
 atom H
 L 0
 exponent 0.2
 value 0.3
end

training
 atom H l
 exp 0.1 0.2
 empty m_empty
 reference m_ref
 subset
  set s22b
 end
end

acp load bleh
 H l 0.1 0.1
 H l 0.2 2000
end

training eval empty
training eval bleh
