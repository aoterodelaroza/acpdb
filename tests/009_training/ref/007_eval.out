* SYSTEM: rm -f 007_eval.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 007_eval.db
Creating skeleton database 

* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_empty

* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_ref

* INSERT: insert data into the database (SET)
# INSERT SET s22b
# INSERT STRUCTURE s22b.nh3_nh3
# INSERT STRUCTURE s22b.nh3_nh3_1
# INSERT STRUCTURE s22b.nh3_nh3_2
# INSERT PROPERTY s22b.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=s22b.nh3_nh3)
# INSERT STRUCTURE s22b.h2o_h2o
# INSERT STRUCTURE s22b.h2o_h2o_1
# INSERT STRUCTURE s22b.h2o_h2o_2
# INSERT PROPERTY s22b.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=s22b.h2o_h2o)

* INSERT: insert data into the database (SET)
# INSERT SET s22b-b
# INSERT STRUCTURE s22b-b.nh3_nh3
# INSERT STRUCTURE s22b-b.nh3_nh3_1
# INSERT STRUCTURE s22b-b.nh3_nh3_2
# INSERT PROPERTY s22b-b.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=s22b-b.nh3_nh3)
# INSERT STRUCTURE s22b-b.h2o_h2o
# INSERT STRUCTURE s22b-b.h2o_h2o_1
# INSERT STRUCTURE s22b-b.h2o_h2o_2
# INSERT PROPERTY s22b-b.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=s22b-b.h2o_h2o)

* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=s22b.nh3_nh3)

* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=s22b.h2o_h2o)

* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.h2o_h2o;atom=H;l=0;exponent=0.1)

* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.h2o_h2o;atom=H;l=0;exponent=0.2)

* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.nh3_nh3;atom=H;l=0;exponent=0.1)

* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=s22b.nh3_nh3;atom=H;l=0;exponent=0.2)

* TRAINING: started defining the training set 

* TRAINING: fininshed defining the training set 

## Description of the training set
# List of atoms and maximum angular momentum channels (1)
| Atom | lmax |
| H | l |

# List of exponents (2)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |

# List of subsets (1)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | s22b | s22b | 1 | 1 | 2 | 2 | 1 |  |  |

# List of methods
| type | name | id | for fit? |
| reference | m_ref | 2 | n/a |
| empty | m_empty | 1 | n/a |

# List of properties (2)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | s22b.nh3_nh3 | 1 | s22b | s22b | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | s22b.h2o_h2o | 2 | s22b | s22b | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |

# Calculation completion for the current training set
# Reference: 2/2 (complete)
# Empty: 2/2 (complete)
# Terms: 
| H | l | 0.1 | 2/2 | (complete)
| H | l | 0.2 | 2/2 | (complete)
# Total terms: 4/4 (complete)

* ACP LOAD bleh

* TRAINING: evaluating the EMPTY method 

# Evaluation: m_empty
# Statistics: 
#   wrms    =  6.25159260
#   wrmsall =  6.25159260 (including evaluation subsets)
# s22b  rms =     4.42054352  mae =     4.31100000  mse =     4.31100000
#  all  rms =     4.42054352  mae =     4.31100000  mse =     4.31100000
Id Name                                      weight             yempty               yacp               yadd             ytotal               yref         difference
1 s22b.nh3_nh3                              1.000000       0.2000000000       0.0000000000       0.0000000000       0.2000000000      -3.1330000000       3.3330000000
2 s22b.h2o_h2o                              1.000000       0.3000000000       0.0000000000       0.0000000000       0.3000000000      -4.9890000000       5.2890000000

* TRAINING: evaluating ACP bleh

# Evaluation: bleh
# Statistics: 
#   2-norm  =  2000.0000025
#   1-norm  =  2000.1000000
#   maxcoef =  2000.0000000
#   wrms    =  603.40738936
#   wrmsall =  603.40738936 (including evaluation subsets)
# s22b  rms =   426.67345683  mae =   305.34100000  mse =   305.34100000
#  all  rms =   426.67345683  mae =   305.34100000  mse =   305.34100000
Id Name                                      weight             yempty               yacp               yadd             ytotal               yref         difference
1 s22b.nh3_nh3                              1.000000       0.2000000000     600.0300000000       0.0000000000     600.2300000000      -3.1330000000     603.3630000000
2 s22b.h2o_h2o                              1.000000       0.3000000000       2.0300000000       0.0000000000       2.3300000000      -4.9890000000       7.3190000000

